docs/manual/network.md

```diff
@@ -0,0 +1,156 @@
+
+## Management network
+As governed by the well-established container [networking principles](https://docs.docker.com/network/) containers are able to get network connectivity using various drivers/methods. The most common networking driver that is enabled by default for docker-managed containers is the [bridge driver](https://docs.docker.com/network/bridge/).
+
+The bridge driver connects containers to a linux bridge interface named `docker0` on most linux operating systems. The containers are then able to communicate with each other and the host via this virtual switch (bridge interface).
+
+In containerlab we follow a similar approach: containers launched by containerlab will be attached with their interface to a containerlab-managed docker network. It's best to be explained by an example which we will base on a [two nodes](../lab-examples/two-srls.md) lab from our catalog:
+
+```yaml
+name: srl02
+
+topology:
+  kinds:
+    srl:
+      type: ixr6
+      image: srlinux
+      license: license.key
+  nodes:
+    srl1:
+      kind: srl
+    srl2:
+      kind: srl
+
+  links:
+    - endpoints: ["srl1:e1-1", "srl2:e1-1"]
+```
+
+As seen from the topology definition file, the lab consists of the two SR Linux nodes which are interconnected via a single point-to-point link.
+
+<div class="mxgraph" style="max-width:100%;border:1px solid transparent;margin:0 auto; display:block;" data-mxgraph="{&quot;page&quot;:3,&quot;zoom&quot;:1.5,&quot;highlight&quot;:&quot;#0000ff&quot;,&quot;nav&quot;:true,&quot;check-visible-state&quot;:true,&quot;resize&quot;:true,&quot;url&quot;:&quot;https://raw.githubusercontent.com/srl-wim/containerlab-diagrams/main/containerlab.drawio&quot;}"></div>
+
+The diagram above shows that these two nodes are not only interconnected between themselves, but also connected to a bridge interface on the lab host. This is driven by the containerlab default management network settings.
+
+### default settings
+When no information about the management network is provided within the topo definition file, containerlab will do the following
+
+1. create, if not already created, a docker network named `clab`
+2. configure the IPv4/6 addressing pertaining to this docker network
+
+!!!info
+    We often refer to `clab` docker network simply as _management network_ since its the network to which management interfaces of the containerized NOS'es are connected.
+
+The addressing information that containerlab will use on this network:
+
+* IPv4: subnet 172.20.20.0/24, gateway 172.20.20.1
+* IPv6: subnet 2001:172:20:20::/80, gateway 2001:172:20:20::1
+
+With these defaults in place, the two containers from this lab will get connected to that management network and will be able to communicate using the IP addresses allocated by docker daemon. The addresses that docker carves out for each container are presented to a user upon the lab deployment finishes or can be queried any time after:
```

Comment:
`upon the lab deployment finishes`
--> once the lab deployment [ finishes | is done ] 
or
--> upon the lab deployment completion